Automattic/wp-calypso
# Desktop App: Enable Gutenberg feature flags

#### Changes proposed in this Pull Request

This enables the `jetpack/gutenframe` and `calypsoify/gutenberg` feature flags in `desktop.json` and `desktop-development.json` in order to support editing flows with the block editor for Jetpack and Atomic sites.

#### Testing instructions

This needs to be tested on the desktop app, so make sure you have the dev environment [installed and running](https://github.com/Automattic/wp-desktop/blob/b729377828ec53f4469ed27a15c1fbc6e9b3a1cf/docs/install.md) (more info about how to set up the secrets in paaHJt-7y-p2).

* In the desktop repo, point the calypso submodule to this branch: `cd calypso && git fetch && git checkout update/enable-gutenframe-desktop-app`.
* <del>Create a build: `make build`.</del>
* <del>Open the built app in the `release` folder.</del>
* Check out the `develop` branch in `wp-desktop`.
* In one tab, run `make dev-server` within the `wp-desktop` folder.
* Once that completes, run `make dev` in a second tab. This should open a dev version of the app.
* Switch to a Jetpack or Atomic site test site running the Jetpack plugin version 7.3.
* Make sure that Gutenberg works like in the production environment for Simple sites:
  * Check opt-in and opt-out flows (making sure that the opt-out flow redirects to the classic editor which is opened inside the desktop app).
  * Try publishing a new post.
  * Test the Gutenframe integration (media modal, revisions, move to trash, etc).
  * Confirm that the back to the posts button redirects to the lists of posts page and it is opened inside the desktop app.
  * Try editing an existing post.
* Switch to a Jetpack or Atomic site test site running a version of the Jetpack plugin older than 7.3.
* Make sure that Calypso uses Calypsoify flows now and the block editor is opened in a new tab of your browser.

Fixes #32048
